src/main/java/org/apache/sysml/runtime/instructions/cp/ListObject.java

```diff
@@ -21,18 +21,20 @@
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.Optional;
 
 import org.apache.sysml.parser.Expression.DataType;
 import org.apache.sysml.parser.Expression.ValueType;
 import org.apache.sysml.runtime.DMLRuntimeException;
+import org.apache.sysml.runtime.controlprogram.caching.CacheableData;
 
 public class ListObject extends Data {
```

Comment:
Please remove all introduced warnings related to raw types and missing switch cases.